### PR TITLE
update chainlink-common

### PR DESCRIPTION
**Pull request by makramkd**
Update chainlink-common to use the family-agnostic types from https://github.com/smartcontractkit/chainlink-common/pull/639.

### CODE REVIEW

File: execute/plugin_test.go [hunk 1]
```diff
@@ -315,8 +316,8 @@ func Test_selectReport(t *testing.T) {
 			},
 			expectedExecReports:   1,
 			expectedCommitReports: 1,
-			expectedExecThings:    []int{5},
-			lastReportExecuted:    []cciptypes.SeqNum{100, 101, 102, 103, 104},
+			expectedExecThings:    []int{8},
+			lastReportExecuted:    []cciptypes.SeqNum{100, 101, 102, 103, 104, 105, 106, 107},
```
Comment (makramkd) [inline execute/plugin_test.go:320]: Not sure why this would change, is it because it marshals to less JSON bytes?

Comment (winder) [reply execute/plugin_test.go:320]: Yes exactly, this is potentially going to be a bit annoying until things settle down.

Could you fix this by adjusting `maxReportSize` instead of adjusting which messages were executed? While this still tests the same edge cases as before, that wont always be the case. Also it would keep the name "half report" accurate. Looks like you figured out that approach for most of the later tests.

Comment (makramkd) [reply execute/plugin_test.go]: Yeah, that makes sense. I've fixed these to only tweak the sizes when needed.

**Review by winder — APPROVED** [review]

**Review by dimkouv — APPROVED** [review]
